tests/py310workaround.py

```diff
@@ -0,0 +1,3 @@
+def fix_python_3_10_compatibility():
```

Comment:
Do we really need this function? It introduces new module, but I cannot see a profit.